### ISSUE TITLE
Support schema on read

### ISSUE THREAD

**Issue by xushiyan** [apache/hudi-rs#77]

**Comment by gohalo** [comment]
does this mean we need to extract the schema from hoodie.properties, instead of the latest parquet file?

**Comment by xushiyan** [comment]
@gohalo not really. it's about implementing https://hudi.apache.org/docs/schema_evolution/#schema-evolution-on-read

**Comment by gohalo** [comment]
just checked how this works, i will work on that
